### PR TITLE
fix url when there is a baseURL

### PR DESCRIPTION
**Pull request by szrenwei**
Change-Id: Ie75cda8c076a2b081b6b05c565b0aea51977d734

fix issue: https://github.com/softonic/axios-retry/issues/67

### CODE REVIEW

**Comment by ivoiv** [comment]
Is this a breaking change since it changes the way baseUrl works right now?

**Comment by szrenwei** [comment]
@ivoiv No, it haven't change baseUrl it self, just fix the config.url, config.url should not include baseUrl.

File: es/index.js [hunk 1]
```diff
@@ -116,6 +116,9 @@ function fixConfig(axios, config) {
   if (axios.defaults.httpsAgent === config.httpsAgent) {
     delete config.httpsAgent;
   }
+  if (config.baseURL) {
+    config.url = config.url.replace(config.baseURL, '');
```
Comment (avindra) [inline es/index.js]: Its more of a workaround really.. As others are commenting on issue #67 , axios really expects an **absolute** url here.

This is also an expectation in the core code of Axios (there are `isAbsoluteUrl` checks before `baseURL` gets applied to anything).

So, the problem is that by using a relative url in `baseURL`, it breaks axios's current expectations.

Comment (mawrkus) [reply es/index.js]: Indeed, it does not look like an axios-retry issue.

**Pull request closed without merging**